### ISSUE TITLE
Cannot import modules in compiled package in another namespace.

### ISSUE THREAD

**Issue by MDTech-us-MAN** [Nuitka/Nuitka#509]
I have the following directory structure:

```
testproject
├── __init__.py
├── testpackage1
│   ├── module1.py
│   ├── module2.py
└── testpackage1
    ├── module1.py
    └── module2.py
```

The `__init__.py` contains simply `print("Loaded project")` and the modules contain something like `print("This is package1, module1")`. I can import each module like so:

```
Python 3.7.4 (tags/v3.7.4:e09359112e, Jul  8 2019, 19:29:22) [MSC v.1916 32 bit
(Intel)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import testproject
Loaded project
>>> import testproject.testpackage1
>>> import testproject.testpackage1.module1
This is package1, module1
>>>
```

After building each testpackage individually and copying the resulting pyd files like so:

```
testproject
├── __init__.py
├── testpackage1.pyd
└── testpackage1.pyd
```

I get the following:

```
Python 3.7.4 (tags/v3.7.4:e09359112e, Jul  8 2019, 19:29:22) [MSC v.1916 32 bit
(Intel)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import testproject
Loaded project
>>> import testproject.testpackage1
>>> import testproject.testpackage1.module1
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'testproject.testpackage1.module1'
```

**Comment by kayhayen** [comment]
You didn't quite say which commands you used to create the `testpackage1.pyd` ?

**Comment by MDTech-us-MAN** [comment]
Just the usual `python -m nuitka - -module testpackage1 - -include-package=testpackage1`.

**Comment by kayhayen** [comment]
When you compile, you create a package `testpackage1`, which is loaded by `testproject` as `testpackage1`, and then registers for `testpackage.module1`.

So the issue is that, while a module is loaded into the name decided at run time, the inclusions below it, are not affected by run time, which is of course wrong.

In our reflected test, where we compile Nuitka with itself in various forms, we only create single modules compiled, but never whole sub-packages.

**Comment by kayhayen** [comment]
I have created a relatively complex change, that makes these included modules dynamic, and changed the loading and handling of loaded modules, to fit and take advantage.

In my testing, this appeared to work now, can you confirm?

**Comment by kayhayen** [comment]
Fixed on factory https://nuitka.net/doc/factory.html

**Comment by kayhayen** [comment]
I am assuming this is fixed, released as part of the rc6 pre-release, will be part of next release.

**Comment by MDTech-us-MAN** [comment]
This is indeed fixed. Thank you very much!

**Comment by kayhayen** [comment]
Released as part of 0.6.6
